tests/legacy_kustomizations/metadata/kustomization.yaml

```diff
@@ -17,7 +17,7 @@ configMapGenerator:
 - behavior: merge
   envs:
   - params_1.env
-  name: metadata-grpc-configmap
+  name: grpc-configmap
```

Comment:
Is this expected? I had to manually fix this file.

Reply:
This doesn't seem to be right. I remember the name has to be `metadata-grpc-configmap`.
cc @dushyanthsc